ios/SecuritySdk/Classes/Digital Signature/Secure Enclave/SecureEnclaveVerifyingKey.swift

```diff
@@ -22,6 +22,17 @@ public struct SecureEnclaveVerifyingKey: VerifyingKey {
     }
 
     // MARK: - Public Class Methods (VerifyingKey)
+    
+    public func exportAsData() throws -> Data {
+        var error: Unmanaged<CFError>?
+        guard let data = SecKeyCopyExternalRepresentation(
+            publicKey,
+            &error
+        ) as? Data else {
+            throw CryptographicKeyError.unexportablePublicKey
```

Comment:
Might be good to add comment here as to why this can fail. According to documentation
`The operation fails if the key is not exportable, for example if it is bound to a smart card or to the Secure Enclave. It also fails in macOS if the key has the attribute kSecKeyExtractable set to false.` https://developer.apple.com/documentation/security/1643698-seckeycopyexternalrepresentation